Docs/doc.html

```diff
@@ -47,10 +47,10 @@ <h1 class="section-header">Introduction</h1>
         <h3 class="section-sub-header">Installation</h3>
         <p>To start using Ozone-UI 
             you just need to import below style sheet in your project</p>
-        <iframe
-            src="https://carbon.now.sh/embed?bg=rgba%28250%2C252%2C254%2C1%29&t=night-owl&wt=none&l=auto&width=696&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14.5px&lh=152%25&si=false&es=2x&wm=false&code=%2540import%2520url%28%2522https%253A%252F%252Fozone-ui.netlify.app%252FDocs%252Fdoc.css%2522%29"
-            style="width: 100%; height: 80px; border:0; transform: scale(1); overflow:hidden;"
-            sandbox="allow-scripts allow-same-origin">
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28112%2C133%2C150%2C1%29&t=night-owl&wt=sharp&l=auto&width=849&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fozone-ui.netlify.app%252FDocs%252Fdoc.css%2522%253E%250A"
+                style="width: 100%; height: 114px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Please try avoid using inline styles. If this is specific in height, you would face difficulty in creating a class for each one separately, you can set height to auto and take the entire style in a class for all iframe styles.

Reply:
sure, Aditi